packages/cli/src/definitions/projectTypes.ts

```diff
@@ -0,0 +1,5 @@
+export enum ProjectTypes {
+    REACT_SCRIPT = "react-scripts",
+    REMIX = "remix",
+    NEXTJS = "nextjs",
+}
```

Comment:
```suggestion
export const projectTypes = "react-scripts" | "remix" | "nextjs"
```
should we use it like this instead of `enum`?